proto/neutron/dex/tx.proto

```diff
@@ -92,11 +92,14 @@ enum LimitOrderType {
 }
 
 message MsgPlaceLimitOrder {
+  // TODO: fix wire numbers
```

Comment:
What exactly do you mean by todo?

Reply:
Just a note to make all the wire numbers sequential at some point. Very low priority